content/docs/tasks/security/authz-tcp/index.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Authorization for TCP services
+description: Shows how to set up role-based access control for TCP services.
+weight: 40
+keywords: [security,access-control,rbac,tcp,authorization]
+---
+
+This task covers the activities you might need to perform to set up Istio authorization, also known
+as Istio Role Based Access Control (RBAC), for TCP services in an Istio mesh. You can read more in
+[authorization](/docs/concepts/security/#authorization) and get started with a basic tutorial in
+Istio Security Basics.
+
+## Before you begin
+
+The activities in this task assume that you:
+
+* Understand [authorization](/docs/concepts/security/#authorization) concepts.
+
+* Have set up Istio on Kubernetes **with authentication enabled** by following the instructions in the
+  [quick start](/docs/setup/kubernetes/quick-start/), this tutorial requires mutual TLS to work. Mutual TLS
+  authentication should be enabled in the [installation steps](/docs/setup/kubernetes/quick-start/#installation-steps).
+
+## Setup Bookinfo with MongoDB TCP service
+
+The normal Bookinfo sample includes only HTTP services by default, in order to demonstrate Istio
+authorization for TCP services, we'll update the Bookinfo sample to use the `v2` of the `ratings`
+service which talks to a MongoDB backend.
+
+> If you are using a namespace other than `default`, use `kubectl -n namespace ...` to specify the namespace.
```

Comment:
Try to keep it consistent. Do not include commands in the text, use a code-block instead. There is no need for the text to be a quote block either. I suggest:

To specify a namespace other than the `default` namespace used in the Bookinfo example, replace [my-namespace] in the following command with the name of your namespace:

{{< text bash >}}
kubectl -n namespace [my-namespace]
{{< /text >}}

Reply:
This is not a command supposed to be used by user, it's suggesting to use the `-n` option to use the actual namespace in user's environment.

I moved it to the `Before you begin` section as a note.